fun-with-categories.md

```diff
@@ -1,32 +1,36 @@
-# 🎉 FUN WITH CATEGORIES 🎉
 
----
+## 🎉 FUN WITH CATEGORIES 🎉
 
-## Marco Perone
+#### Marco Perone
 
-[twitter.com/marcoshuttle](https://twitter.com/marcoshuttle)
+![statebox](img/statebox_simpleblack_txt_bottom.png)
```

Comment:
I think that the talk should be focused on its topic, which is not Statebox itself, and in this case I'd prefer to reserve a slide for Statebox

Reply:
Well, Statebox is about applying category theory and we present our category theory library here. You do the presenting because you worked a lot on this library, but it was a collaborative Statebox effort. At this stage we really want to take every opportunity to promote the project and build our community, this is the main reason why we apply for talks.